kbfsblock/bserver_errors.go

```diff
@@ -1,375 +0,0 @@
-// Copyright 2016 Keybase Inc. All rights reserved.
-// Use of this source code is governed by a BSD
-// license that can be found in the LICENSE file.
-
-package kbfsblock
-
-import (
-	"errors"
-	"fmt"
-	"strconv"
-
-	"github.com/keybase/client/go/libkb"
-	"github.com/keybase/client/go/protocol/keybase1"
-	"github.com/keybase/go-framed-msgpack-rpc/rpc"
-)
-
-const (
-	// StatusCodeBServerError is the error code for a generic block server error.
-	StatusCodeBServerError = 2700
```

Comment:
I don't know why github doesn't detect this as a rename, but I literally did `s/BServerError/ServerError/` in this file.